configure

```diff
@@ -0,0 +1,4 @@
+#! /bin/bash
```

Comment:
configure should not be part of this. Once PBUILD is modified, we won't need that anymore. Please remove once your testing is complete. PBUILD change will follow it shortly.